journal-letters/modernized/03-chapter-1.adoc

```diff
@@ -222,17 +222,17 @@ who in the joy of their souls broke forth in praises unto the Lord;
 so that the tongue of the dumb, which Christ the healer of our infirmities did unloose,
 began to speak and utter the wonderful things of God.
 Great was the dread and glory of that power,
-which in one meeting after another was graciously and richly manifested amongst us,
+which in one meeting after another was graciously and richly manifested among us,
 breaking, tendering, and melting our souls and spirits before the Lord.
 Then our hearts began to delight in the Lord and in his way that he had cast up;
 and with great fervency and zeal we began to seek after him,
-and to meet oftener together than before,-- our hearts
+and to meet together more often than before,-- our hearts
 being affected with the presence of that blessed power,
-which daily broke forth amongst us in our meetings,
+which daily broke forth among us in our meetings,
 through which we were greatly comforted, strengthened and edified;
 for it was that same Comforter our blessed Lord promised he would pray the Father for,
 and which the Father should send.
-John 14:16 and 26 This Comforter being come and received,
+John 14:16 and 16:26. This Comforter being come and received,
```

Comment:
Here he actually means John 14: verses 16 and 26. But you were right to add the period after the 26.